packages/web-client/src/webdav/getFileUrl.ts

```diff
@@ -8,18 +8,22 @@ import { DAV } from './client'
 export const GetFileUrlFactory = (
   dav: DAV,
   getFileContentsFactory: ReturnType<typeof GetFileContentsFactory>,
-  { sdk, capabilities }: WebDavOptions
+  { sdk, capabilities, clientService, user }: WebDavOptions
 ) => {
   return {
     async getFileUrl(
       space: SpaceResource,
       resource: Resource,
       {
         disposition = 'attachment',
-        signUrlTimeout = 86400
+        signUrlTimeout = 86400,
+        version = null,
+        preflightHeadRequest = false
```

Comment:
I find the var name a bit confusing. `preflight` is usually an `OPTIONS` request, not `HEAD`. Maybe `doHeadRequest` or something similar would be more accurate? What do you think?

Reply:
No hard feelings here, I'm fine with `doHeadRequest` 👍